dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
+            )
+
+            app$run_server()
+    - name: htmlMarquee
+      dontrun: TRUE
+      code: |
+            # This feature is obsolete. It may still work in some
+            # browsers, but could stop working at any time. Try to
+            # avoid using this component.
+
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlMarquee("Here is some sliding text that uses htmlMarquee")
+
+              ))
+            )
+
+            app$run_server()
+    - name: htmlMeta
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlP("The <meta> tag provides metadata about the HTML document.
+                      Metadata will not be displayed on the page, but will be machine parsable.
+                      To view meta tag the content of this page can be inspected."),
+                htmlMeta(name = "author", content = "Edward Tufte")
+              ))
+            )
+
+            app$run_server()
+    - name: htmlMeter
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlLabel("Sample Level Meter: "),
+                htmlMeter(id = "sample-meter",
+                          min = 0,
+                          max = 100,
+                          low = 33,
+                          high = 66,
+                          optimum = 80,
+                          value = 80
+                          )
+              ))
+            )
+
+            app$run_server()
+    - name: htmlMulticol
+      dontrun: TRUE
+      code: |
+            # Warning: The <multicol> tag is obsolete, it might not work as intended.
+            # Try to avoid using it.
+    - name: htmlNav
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            library(dashCoreComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlNav(
+                  list(
+                    htmlA("Plotly", href = "https://plot.ly/"),
+                    '> ",
+                    htmlA("Dash", href = "https://plot.ly/dash"),
+                    '> ",
+                    htmlA("Request Trial", href = "https://go.plot.ly/dash-enterprise-trial")
+                  )
+                )
+              ))
+            )
+
+            app$run_server()
+    - name: htmlNextid
+      dontrun: TRUE
+      code: |
+            # Warning: The <nextid> tag is obsolete, as of HTML Version 3.2.
```

Comment:
then take out the code below, which doesn't use `htmlNextid` at all?

Reply:
fixed in 5555dff